python/lsst/resources/http.py

```diff
@@ -74,7 +75,8 @@ def _is_webdav_endpoint(path: Union[ResourcePath, str]) -> bool:
     _is_webdav_endpoint : `bool`
         True if the endpoint implements WebDAV, False if it doesn't.
     """
-    if (ca_cert_bundle := os.getenv("LSST_HTTP_CACERT_BUNDLE")) is None:
+    is_https = str(path).lower().startswith("https://")
+    if (ca_cert_bundle := os.getenv("LSST_HTTP_CACERT_BUNDLE")) is None and is_https:
```

Comment:
Fixed. The intention was to give some hints to the user in case of authentication errors. Hints are provided when actual authentication errors are found.